app/src/main/java/com/kylecombes/micrepeater/BluetoothStateReceiver.java

```diff
@@ -31,6 +28,9 @@ public void registerStateChangeReceiver(StateChangeReceiver receiver) {
 
     @Override
     public void onReceive(Context context, Intent intent) {
+        boolean mScoAudioConnected = false;
+        boolean mDeviceConnected;
```

Comment:
I think variables starting with `m` is for private variables inside a class. This is a temp variable so no need the `m`.